src/core/src/main/resources/org/apache/jmeter/resources/messages_fr.properties

```diff
@@ -15,15 +15,15 @@
 
 #Stored by I18NEdit, may be edited!
 about=A propos de JMeter
-action_check_message=Un test est en cours, arrêtez le avant d''utiliser cette commande
+action_check_message=Un test est en cours, arr\u00eatez le avant d''utiliser cette commande
```

Comment:
Check your build, since gradle migration file is in UTF-8

Reply:
This has not been taken into account, can you please check ? There are too much differences